index.html

```diff
@@ -8,8 +8,144 @@
   <link rel="stylesheet" href="styles.css">
   <title>Document</title>
 </head>
+
 <body>
+  <header class="mobileHeader">
+    <nav id="mobMenuPH">
+      <input type="image" id="mobileMenuIndx" src="images/mobile_menu.png" alt="Menu">
+    </nav>
+    <a href="index.html">
+      <img id="mobLogo" src="images/mobileMenu.svg" alt="logo" alt="Logo">
+    </a>
+  </header>
+  <header class="deskHeader">
+    <div class="topMenu1">
+      <a class="facebook" href="https://www.facebook.com/">
+        <img src="images\facebook.png" alt="Twitter">
+      </a>
+      <a class="twitter" href="https://twitter.com/ricmtza">
+        <img src="images\twitter.png" alt="Twitter">
+      </a>
+      <a class="english topMenuTxt" href="index.html">English</a>
+      <a class="myPage topMenuTxt" href="index.html">My Page</a>
+      <a class="logout topMenuTxt" href="index.html">Logout</a>
+    </div>
+    <div class="topMenu2">
+      <a class="deskImgPH" href="index.html">
+        <img class="deskImg" src="images/mobileMenu.svg" alt="CC Logo">
+      </a>
+      <a class="topMenu2Txt" href="html/about.html">About</a>
+      <a class="topMenu2Txt" href="index.html">Program</a>
+      <a class="topMenu2Txt" href="index.html">Join</a>
+      <a class="topMenu2Txt" href="index.html">Sponsor</a>
+      <a class="topMenu2Txt" href="index.html">News</a>
+      <a class="topMenu2Txt" href="index.html">CC Campaign</a>
+      
+    </div>

```

Comment:
It would be nice if your add an orange border arount the CC campaign just like it is in the design. Please pa attention to detail.

Here is your design: 

![nav ](https://user-images.githubusercontent.com/79506105/192030284-2a9f31c9-7509-45c5-9304-1a905b32a354.png)

Your design should look like this 

![Screen Shot 2022-09-23 at 17 47 47](https://user-images.githubusercontent.com/79506105/192030368-5bf01fa7-bf2e-4efb-b92e-b916420e6cbb.png)